### PR TITLE
Nert 689 - Downloading masks from all list exports

### PR DESCRIPTION
**Pull request by popkinj**
- Public and Non-Public list pages (projects and plans) now send the mask instead of the original geometry (when specified).
- The project/plan centroids now disclaim the level of accuracy. 
Screenshot from 2024-10-02 11-14-44

### CODE REVIEW

**Comment by popkinj** [comment]
I'm guessing this is failing due to the same issue we had before.... empty geometries??

File: app/src/components/map/MapContainer.tsx [hunk 1]
```diff
@@ -1053,6 +1053,7 @@ const initializeMap = (
           size_ha={sizeHa}
           state_code={stateCode}
           thumbnail={thumbnail}
+          maskDisclaimer={true}
```
Comment (KjartanE) [inline app/src/components/map/MapContainer.tsx:1056]: Only added for Project? Should add `true` to Plan as well `markerPlans.points`

File: api/src/paths/public/plans.ts [hunk 2]
```diff
@@ -354,6 +355,15 @@ export function getPublicPlansList(): RequestHandler {
       // Get all projects data for the projectIds
       const projects = await planService.getPlansByIds(projectIds, true);
 
+      // Mask geometries that require it
+      projects.forEach((project) => {
```
Comment (KjartanE) [inline api/src/paths/public/plans.ts:359]: You broke the tests for each of these endpoints, could you update them accordingly?

File: app/src/components/map/MapContainer.tsx (hunk 1 above)
Comment (popkinj) [reply app/src/components/map/MapContainer.tsx:1056]: Good catch.

Screenshot from 2024-10-03 14-56-04

File: api/src/paths/public/plans.ts (hunk 2 above)
Comment (popkinj) [reply api/src/paths/public/plans.ts:359]: Fixed now.

**Review by KjartanE — APPROVED** [review]
🐈

**Pull request merged**